install-internal-package/action.yml

```
@@ -13,29 +13,42 @@ inputs:
 
 runs:
   using: composite
+  defaults:
+    run:
+      shell: bash
+      working-directory: ${{ inputs.working-directory }}
   steps:
     - name: Authenticate to GitHub Packages
-      shell: bash
       run: npm set //npm.pkg.github.com/:_authToken="${{ github.token }}"
-      working-directory: ${{ inputs.working-directory }}
 
     - name: Determine DevExtreme Version
       id: devextreme_version
-      shell: bash
       run: echo "version=$(npm pkg get dependencies.devextreme --workspaces=false | tr -d '"')" >> $GITHUB_OUTPUT
-      working-directory: ${{ inputs.working-directory }}
 
-    - name: Install Internal devextreme Package
-      shell: bash
+    - name: Download Internal devextreme Package
+      id: download_devextreme_package
       run: |
         npm pack @devexpress/devextreme-internal@${{ steps.devextreme_version.outputs.version }} --registry https://npm.pkg.github.com
-        npm i devextreme@$(find . -name DevExpress-devextreme-internal-*.tgz)
-      working-directory: ${{ inputs.working-directory }}
+        echo "dxtfilename=$(find . -name devexpress-devextreme-internal-*.tgz)" >> $GITHUB_OUTPUT
```

It seems to be enough specific because of the step id
```suggestion
        echo "packageFile=$(find . -name devexpress-devextreme-internal-*.tgz)" >> $GITHUB_OUTPUT
```